packages/bun/src/sdk.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable max-lines */
+import { Integrations as CoreIntegrations } from '@sentry/core';
+import { init as initNode, Integrations as NodeIntegrations } from '@sentry/node';
+
+import { BunClient } from './client';
+import { makeFetchTransport } from './transports';
+import type { BunOptions } from './types';
+
+export const defaultIntegrations = [
+  // Common
+  new CoreIntegrations.InboundFilters(),
+  new CoreIntegrations.FunctionToString(),
+  // Native Wrappers
+  new NodeIntegrations.Console(),
+  new NodeIntegrations.Http(),
+  new NodeIntegrations.Undici(),
+  // Global Handlers # TODO
```

Comment:
l: just for some extra context:

```suggestion
  // Global Handlers # TODO (waiting for https://github.com/oven-sh/bun/issues/5091)
```